config/daemons.yaml

```diff
@@ -11,6 +11,7 @@ daemons:
     serviceAccount:
       name: nnf-dm-daemon
       namespace: nnf-dm-system
+    extraArgs: "--kubernetes-qps 5 --kubernetes-burst 10"
```

Comment:
Didn't you determine those values are too small?

Reply:
Whoops. Forgot to switch these back after testing.